Emmy-github-webdev/my-portfolio
# Portfolio: Accessibility check

## Portfolio: Accessibility check.

In the accessibility check I:

- Test Page title: No issue found
- Test Image text alternative ** (<img alt="alt text"): Give distinct text to all the image text alternative
- Check Text headings: No issue found
- Check Color contrast: Update h2 for social media color to pass accessibility test
- Check Resize: No issue found
- Check Interaction: No issue found
- Check Moving content: No issue found
- Check Multimedia: No issue found
- The basic structure of the page: No issue found
